Symfony/CS/Fixer/Symfony/PhpdocNoPackageFixer.php

```diff
@@ -39,4 +39,15 @@ public function getDescription()
     {
         return '@package and @subpackage annotations should be omitted from phpdocs.';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        /*
+         * Needs higher priority than any of the fixers cleaning up empty docblocks or fixing headers.
```

Comment:
This is also not correct. The `@package` stripper specifically can leave an empty phpdoc behind, so the correct order is:
 - Strip `@package`
 - Convert to regular comment
 - Let the `header comment` fixer do its job (in this case removing it)